### ISSUE TITLE
[Feature Request]: Show color palettes & add more

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-issues#2122]
### Description

Show color palettes & names that mean something (like color blind) in drop-down menu & add some more palettes

### Purpose

to see beforehand, what you are selecting & to have more appropriate palettes for heat maps

### Use-case

seeing the color in the drop down is better for non-technical users that do not know color palette names like ggplot2 or viridis; for heat maps a color palett from blue over white to red is often a good choice

### Is your feature request related to a problem?

selecting the right palette is hard atm

### Is your feature request related to a JASP module?

Descriptives

### Describe the solution you would like

see description

### Describe alternatives that you have considered

-

**Comment by tomtomme** [comment]
relevant 2023 paper on the new color palettes in R since 4.0
https://www.zeileis.org/news/coloring/

**Comment by tomtomme** [comment]
Great! However I cannot yet select those two new color palletes in the descriptives module.

**Comment by tomtomme** [comment]
Forget my former comment. I just saw that there is a bugfix for this in 0.18.1 :D

**Comment by tomtomme** [comment]
0.18.1 includes the new palletes, but still does not show them in the dropdown menu, just the names

**Comment by tomtomme** [comment]
still valid for 0.18.2 beta from 30.11.23

**Comment by tomtomme** [comment]
a nice R-Package to include more palettes: cols4all https://github.com/mtennekes/cols4all

**Comment by tomtomme** [comment]
from vandenman:
However, the problem is a bit more complex than just one general option. Take for example this plot:

![Image](https://github.com/user-attachments/assets/7c08262b-af00-487f-a825-ba1c776c4b11)

This color palette is meant for a discrete scale, but it's used as a gradient.

I think we need multiple color palettes. One palette for colors that represent discrete scales (e.g., 4 different lines or points with different colors [example](https://ggplot2.tidyverse.org/reference/geom_point.html)) and one palette for colors that represent continuous scales (e.g., a bivariate density plot [example](https://ggplot2.tidyverse.org/reference/geom_contour.html)).

Usually, colors palettes meant for continuous scales work fine for discrete scales, but not the other way around.

**Comment by vinniott** [comment]
I remember that @vandenman suggested (somewhere, cant find it ad-hoc) that any changes in palettes names should also be documented somewhere so that users can look up the new name of the [former palette name] that they are looking for.

How and where do you think this is best documented? Because the palette drop-down itself probably shouldnt hold all that information, right? Similarly, placing this in every help file for each analysis would also be too much, right?

**Comment by tomtomme** [comment]
Correct. We could keep the old names and use a Preview of the pallette in addition within the dropdown? 
But this could get crowded fast. Maybe additional pallettes will need an extra pop up

**Comment by tomtomme** [comment]
@vincentott 
Just linking your earlier work on this here: https://github.com/jasp-stats/jasp-desktop/pull/5986
